frontend/src/Context/user.tsx

```diff
@@ -49,15 +50,17 @@ export type UserProviderProps = {
 const UserProvider = ({ children }: UserProviderProps): ReactElement => {
   // Load initial settings from local storage
   const noiseSuppression = getStorageItem(STORAGE_KEYS.NOISE_SUPPRESSION) === 'true';
-  const blur = getStorageItem(STORAGE_KEYS.BACKGROUND_BLUR) === 'true';
+  const backgroundFilter = getStorageItem(STORAGE_KEYS.BACKGROUND_REPLACEMENT);
```

Comment:
How do you feel about adding some validation or a more robust parser for this item? We have something similar with `getInitialBackgroundFilter` already. As-is, this feels a little naive to assume the stored item is a valid filter.